app/components/bible-reader/BibleReader.tsx

```diff
@@ -0,0 +1,56 @@
+import { BookOpenIcon } from '@heroicons/react/24/solid';
+import { useEffect, useState } from 'react';
+
+export interface Passage {
+  text: string;
+  verse: string;
+}
+
+const RandomBibleReader = ({ className }: { className?: string }) => {
```

Comment:
i might be worth putting it outside the component, so we avoid reassignment. I suggest place it on a constants.ts file